client.go

```diff
@@ -419,18 +422,25 @@ func (c *Client) UnlockCommits(vsys, admin string) error {
 // Commits result in a job being submitted to the backend.  The job ID and
 // if an error was encountered or not are returned from this function.  If
 // the job ID returned is 0, then no commit was needed.
-func (c *Client) Commit(desc string, dan, pao, force, sync bool) (uint, error) {
+func (c *Client) Commit(desc string, admins []string, dan, pao, force, sync bool) (uint, error) {
     c.LogAction("(commit) %q", desc)
 
     req := baseCommit{Description: desc}
-    if !dan || !pao {
+    if len(admins) > 0 || !dan || !pao {
         req.Partial = &baseCommitPartial{}
         if !dan {
             req.Partial.Dan = "excluded"
         }
         if !pao {
             req.Partial.Pao = "excluded"
         }
+	if len(admins) > 0 {
```

Comment:
You can just replace this whole code block (line 437 through 443) with this:

```golang
req.Partial.Admin = util.StrToMem(admins)
```

Reply:
oh.. sorry! I didn't notice this util.. I have updated.